pkg/executor/containerexecutor/tmpl.go

```diff
@@ -210,7 +210,7 @@ func NewJobOptions(log *zap.SugaredLogger, templatesClient templatesv1.Interface
 	execution testkube.Execution, options client.ExecuteOptions, natsUri string, debug bool) (*JobOptions, error) {
 	jobOptions := NewJobOptionsFromExecutionOptions(options)
 	if execution.PreRunScript != "" || execution.PostRunScript != "" {
-		jobOptions.Command = []string{filepath.Join(executor.VolumeDir, EntrypointScriptName)}
+		jobOptions.Command = []string{"source", filepath.Join(executor.VolumeDir, EntrypointScriptName)}
```

Comment:
Isn't `Command` directly a `command` in the container? If so, we shouldn't have `source` here, as it's not executable - it's built-in function in shell:
```
$ type source
source is a shell builtin
```

Reply:
Yes, I was goiny to check